### PR TITLE
[Needs review] Many additions and changes to the docs

### PR DESCRIPTION
**Pull request by pieterlexis**

### CODE REVIEW

File: docs/manpages/pdns_control.1.md [hunk 1]
```diff
@@ -72,10 +74,10 @@ ping, rping
 purge [*RECORD*]
 :    Purge entries from the packet cache. If *RECORD* ends with a dollar ($)
      all entries that end with that name are removed. If no record is specified
-     the entire cache is purged.
+     the entire packet cache is purged.
```
Comment (ahupowerdns) [inline docs/manpages/pdns_control.1.md]: sure we don't also clear the query cache?

Comment (pieterlexis) [inline docs/manpages/pdns_control.1.md]: If I read `DLPurgeHandler` in pdns/dynhandler.cc correctly (https://github.com/PowerDNS/pdns/blob/master/pdns/dynhandler.cc#L122), we don't touch the querycache on a purge command.

Comment (Habbie) [inline docs/manpages/pdns_control.1.md]: The query cache lives in the same structure and thus `purge` clears them both.

**Comment by Habbie** [comment]
Merge conflict. Did not review yet.

**Comment by Habbie** [comment]
additional doc nit: our export notes do not mention OpenSSL yet

**Pull request merged**